include/intersectables/plane.h

```diff
@@ -32,5 +33,10 @@ class Plane : public Intersectable {
   // f(p(t)) = 0. Solve for t.  Plug t_i into
   // p(t_i) will give intersection point
   virtual HitRecord* intersect(const Ray& ray) const;
+
+  BoundingBox getBoundingBox() const {
+    BoundingBox foo(Vector3f(0, 0, 0), Vector3f(0, 0, 0));
```

Comment:
dummy implementation. Dito, I'm not sure what to return here since the plane itself is the bounding box.